generated/lib.es5.d.ts

```diff
@@ -3843,31 +4190,37 @@ interface Int16ArrayConstructor {
    * @param items A set of elements to include in the new array object.
    */
   of(...items: number[]): Int16Array;
-
   /**
    * Creates an array from an array-like or iterable object.
-   * @param arrayLike An array-like or iterable object to convert to an array.
+   * @param source An array-like or iterable object to convert to an array.
    */
-  from(arrayLike: ArrayLike<number>): Int16Array;
-
+  from(source: ArrayLike<number>): Int16Array;
   /**
    * Creates an array from an array-like or iterable object.
-   * @param arrayLike An array-like or iterable object to convert to an array.
+   * @param source An array-like or iterable object to convert to an array.
    * @param mapfn A mapping function to call on every element of the array.
    * @param thisArg Value of 'this' used to invoke the mapfn.
    */
-  from<T>(
-    arrayLike: ArrayLike<T>,
-    mapfn: (v: T, k: number) => number,
-    thisArg?: any
+  from<T, This = undefined>(
+    source: ArrayLike<T>,
+    mapfn: (this: This, v: T, k: number) => number,
+    thisArg?: This
   ): Int16Array;
 }
-declare var Int16Array: Int16ArrayConstructor;
+//     /**
+//      * Creates an array from an array-like or iterable object.
+//      * @param arrayLike An array-like or iterable object to convert to an array.
+//      */
+//     from(arrayLike: ArrayLike<number>): Int16Array;
+//     /**
+//      * Creates an array from an array-like or iterable object.
+//      * @param arrayLike An array-like or iterable object to convert to an array.
+//      * @param mapfn A mapping function to call on every element of the array.
+//      * @param thisArg Value of 'this' used to invoke the mapfn.
+//      */
+//     from<T>(arrayLike: ArrayLike<T>, mapfn: (v: T, k: number) => number, thisArg?: any): Int16Array;
 
-/**
- * A typed array of 16-bit unsigned integer values. The contents are initialized to 0. If the
- * requested number of bytes could not be allocated an exception is raised.
- */
```

Comment:
Thank you for this! :smiley:
The aliasing system seems right, but one regrettable thing is that we lose those doc comments.
Do you have idea on recovering them?

Reply:
Yea I did notice that, and it seems the the problem already exists before the PR because comments are not printed in the `printInterface` function.

Reply:
oops 🤕